CMakeLists.txt

```diff
@@ -61,10 +61,10 @@ endif()
 
 # Configure a setup.sh
 if(ROBOTOLOGY_USES_GAZEBO)
-  if(WIN32)
-    # On Windows we assume that the Gazebo's enviroment variables are already set,
+  if(WIN32 OR ROBOTOLOGY_CONFIGURING_UNDER_CONDA)
+    # On Windows or Conda we assume that the Gazebo's enviroment variables are already set,
     # so we set as this name a file that does not exists and so will be ignored
-    set(GAZEBO_SETUP_SH_PATH "on-windows-we-assume-that-gazebo-env-variables-are-already-set")
+    set(GAZEBO_SETUP_SH_PATH "on-windows-or-conda-we-assume-that-gazebo-env-variables-are-already-set")
```

Comment:
Side note: I guess that you've chosen to set a message string here instead of leaving the variable empty because the user gets a "warning" if he tries to use the variable?

Reply:
Yes, that was the idea.